### PR TITLE
Initial version of DRF utils extracted from ployst.

### PR DESCRIPTION
**Pull request by txels**
- Serializer mixins for filtering and expanding
- auto404 decorator
- Gravatar field

@alexcouper 

TODO:
- [x] Add tests

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+from setuptools import setup
+import os
+
+__version__ = '0.0.0'
```
Comment (txels) [inline setup.py]: I've cheekily published a 0.0.0 version to pypi already

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+from setuptools import setup
+import os
+
+__version__ = '0.0.1'
```
Comment (txels) [inline setup.py]: This has already been cheekily uploaded to pypi.

File: drfutils/decorators.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+from functools import wraps
+from django.core.exceptions import ObjectDoesNotExist
+from django.http import Http404
+
+
+def auto404(fun):
```
Comment (alexcouper) [inline drfutils/decorators.py:6]: makes lots of sense to have these here

**Comment by txels** [comment]
@alexcouper this now has tests

**Comment by alexcouper** [comment]
Looks good!

**Pull request merged**